modules/nf-core/diamond/blastp/tests/tags.yml

```diff
@@ -0,0 +1,2 @@
+diamond/blastp:
+  - modules/nf-core/diamond/blastp/**
```

Comment:
```suggestion
  - modules/nf-core/diamond/blastp/**
  - modules/nf-core/diamond/makedb/**
```

Reply:
These get autogenerated now, I think anyway. It also doesn't matter because the tag from the test file gets picked up, this just tells the changes files what tags to use.

Reply:
Added the chained module

Reply:
Okay, we'll see if this breaks anything!